pallets/market/src/tests.rs

```diff
@@ -0,0 +1 @@
+
```

Comment:
ups?

Reply:
it is a feature so all the tests pass :D